### PR TITLE
Proposal for Transaction Signing with Albedo on Stellar Laboratory

### PR DESCRIPTION
**Pull request by ralphilius**

### CODE REVIEW

File: bounties/level-2/stellar-lab-albedo.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Sign with Albedo on Stellar Laboratory
+
+| Hunter | Agent
+| :- | :-
+| 1500 XLM | 6 Credits
```
Comment (tyvdh) [inline bounties/level-2/stellar-lab-albedo.md:5]: Let's bump this to 2000 XLM, this is an important PR with high value add imo

File: bounties/level-2/stellar-lab-albedo.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+ - [ ] Code Quality - Is the code well-organized and documented? Does it adhere to language-specific style guidelines?
+ - [ ] UI/UX - Is it comprehensive to users? Is it aligned with current design of Laboratory?
+
+#### *Submission Procedure*
+
+ - Fork the laboratory [repository](https://github.com/stellar/laboratory) with implementing the solution
+ - Create a [new issue](https://github.com/tyvdh/stellar-quest-bounties/issues/new) in the bounty [repository](https://github.com/tyvdh/stellar-quest-bounties)
+   - Choose `review <title of the bounty issue>` as the issue's title
+   - In the new issue description link to the original bounty issue and your fork
+   - Add a comment to the bounty issue linking to your newly created review issue to denote it being ready for review
```
Comment (tyvdh) [inline bounties/level-2/stellar-lab-albedo.md:33]: Also include a requirement for a pending PR in the Lab repo

**Review by tyvdh — CHANGES_REQUESTED** [review]
Can we get a screenshot of the current signing options just to help orient folks to where changes will be taking place in the UI?

File: bounties/level-2/stellar-lab-albedo.md (hunk 2 above)
Comment (ralphilius) [reply bounties/level-2/stellar-lab-albedo.md:36]: I've added the PR requirement

**Comment by ralphilius** [comment]
> Can we get a screenshot of the current signing options just to help orient folks to where changes will be taking place in the UI?

Yes. I've attached current UI.

**Review by tyvdh — APPROVED** [review]

**Pull request merged**